File: src/test/java/com/example/whykotlin/java/nullability/LibrarianTest.java
package com.example.whykotlin.java.nullability;

import org.junit.Test;

public class LibrarianTest {

    @Test
    public void name() {
        Librarian librarian = new Librarian();

        Book book = new Book(null, 123);

        librarian.getTitleLengthOfBook(book);
    }
}

File: src/test/java/com/example/whykotlin/kotlin/nullability/LibrarianTest.kt
package com.example.whykotlin.kotlin.nullability

import org.junit.Test

class LibrarianTest {

    @Test
    fun test() {

        val librarian = Librarian()

        val book = Book("moby dick", 123)

        librarian.getTitleLengthOfBook(book)
    }
}